awx/plugins/isolated/mkfifo.py

```diff
@@ -16,7 +22,10 @@ def main():
     path = module.params['path']
     os.chmod(path, stat.S_IRUSR | stat.S_IWUSR)
     with open(path, 'w') as fifo:
-        fifo.write(module.params['content'])
+        data = module.params['content']
+        if 'OPENSSH PRIVATE KEY' in data and not data.endswith('\n'):
```

Comment:
`We need this check because when we read the FIFO in the run_isolated.yml playbook via lookup('pipe', ...) the ansible pipe lookup plugins strips all trailing whitespace`